src/components/modals/PrivacySettingsModal.vue

```diff
@@ -105,15 +142,20 @@ export default {
   data() {
     return {
       modalVisible: false,
+      newWhitelistedEmail: null,
       newWhitelistedAddress: null,
+      showEthereumAddressField: true,
```

Comment:
Let's simplify this logic a bit by always showing email address first if it's supported regardless of what type of account the user has. That let's you get rid of the `mounted` hook and the new logic inside of `toggleWhitelistField`.

Reply:
Eh, I dunno I feel like it makes sense for savvy users to see addresses instead of emails by default. Besides, I would still need the logic in `mounted` and `toggleWhitelistField` since those are checking `supportEmailAccounts`. The only special logic for savvy users is the first half of the conditional in `mounted`.

Reply:
I removed the conditional in `toggleWhitelistField()`, but I think it makes sense to leave the logic in `mounted` since I think it makes more sense for savvy users to default to the address field.